src/app/apps/rup/mapa-camas/sidebar/cama-detalle/cama-detalle.component.ts

```diff
@@ -83,6 +90,19 @@ export class CamaDetalleComponent implements OnInit {
             pluck('acciones'),
             map(acciones => acciones.filter(acc => acc.tipo === 'nuevo-registro'))
         );
+
+        this.hayMovimientosAt$ = combineLatest(
+            this.camaSelectedSegunView$,
+            this.mapaCamasService.historialInternacion$,
+        ).pipe(
+            map(([cama, historial]) => {
+                if (cama.extras && cama.extras.ingreso) {
+                    return historial.length === 1;
```

Comment:
Si en el historial de la internación llega el mismo movimiento no es necesario chequear el snapshot actual. 

```suggestion
                    return historial.length === 1 && historial[0].extras.ingrso;
```